lib/juttle-engine.js

```diff
@@ -1,45 +1,28 @@
-var Base = require('extendable-base');
+'use strict';
 var _ = require('underscore');
 var express = require('express');
 var expressWs = require('express-ws');
-var bodyParser = require('body-parser');
-var jobs = require('./job-handlers');
-var paths = require('./path-handlers');
-var prepares = require('./prepare-handlers');
 var logger = require('log4js').getLogger('juttle-engine');
 
 var read_config = require('juttle/lib/config/read-config');
 var Juttle = require('juttle/lib/runtime').Juttle;
-var JuttledErrors = require('./errors');
+var JuttleExpressRoutes = require('./express-routes');
```

Comment:
Naming nit -- these routes aren't actually just juttle routes and it's self-evident they're express routes, so my preference would be to rename the file to be `routes.js` and change this to just simply `var add_routes = require('./routes')`;